csi/node_server.go

```diff
@@ -800,11 +800,6 @@ func (ns *NodeServer) NodeExpandVolume(ctx context.Context, req *csi.NodeExpandV
 		return nil, status.Error(codes.InvalidArgument, "volume id missing in request")
 	}
 
-	if req.VolumeCapability.GetBlock() != nil {
```

Comment:
From the code change, does this mean encryption volume expansion never worked before? @mantissahz